phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/ui/LeftButton.java

```diff
@@ -29,7 +29,7 @@ public LeftButton(String name) {
         super(name);
         this.name = name;
         this.getStyleClass().add("leftButton");
-        this.setPrefWidth(Double.MAX_VALUE);
+        this.setMaxWidth(Double.MAX_VALUE);
```

Comment:
If it's not set, the buttons have different width. Is it possible to set it in CSS (how to say `Double.MAX_VALUE`)?